packaging/datadog-agent/source/supervisor.conf

```diff
@@ -19,7 +19,6 @@ serverurl = unix://%(here)s/../run/agent-supervisor.sock
 
 [program:collector]
 command=python agent/agent.py foreground --use-local-forwarder
-stdout_logfile=supervisord/logs/collector.log
```

Comment:
Looks like we are just removing that without replacing that and we are ending without any logs.